10up/maps-block-apple
# Update phpcs installer plugin, allow to run.

### Description of the Change


Updates phpcodesniffer-composer-installer version to resolve a conflict during install.

> **Note**
> The WPCS action failing is due to another issue, see PR #144


Closes #142 

### How to test the Change


1. Checkout branch
2. Run `composer install`
3. Ensure the dependencies install as expected
4. Ensure PHP CodeSniffer Config `installed_paths set` to `../../wp-coding-standards/wpcs`

### Changelog Entry

> Changed - Update composer dependencies to avoid conflict.



### Credits

Props @peterwilsoncc 


### Checklist:
